docs/contrib/contributing.rst

```diff
@@ -89,18 +89,32 @@ In order for a Pull Request to be considered, it has to meet these requirements:
 
 2.  Be accompanied by appropriate tests.
 
-3.  Be issued from a branch *other than master* (PRs coming from master will not
+3.  Be accompanied by appropriate documentation
+    :ref:`documentation <project-documentation>`.
+
+4.  Be issued from a branch *other than master* (PRs coming from master will not
     be accepted.)
 
 If not *all* of these requirements are met then the code should **not** be
 merged into the distribution, and need not even be reviewed.
 
+
+.. _project-documentation:
+
 Documentation
 -------------
 
-All code contributed to the user-facing standard library (the `nativelib/`
-directory) should come accompanied with documentation.
-Pull requests containing undocumented code will not be accepted.
+Pull requests, other than the most trivial, without documentation
+will not be accepted. Imagine yourself as the person merging trying
```

Comment:
Suggestion:
```suggestion
Pull requests without documentation will not be accepted,
unless they are trivial. Imagine yourself as the person merging trying
```

Reply:
Fixed. Suggested wording is much better.